{
    "has_execution_error": true,
    "execution_output": "Model reinitialization failed: CUDA out of memory. Tried to allocate 270.00 MiB. GPU 1 has a total capacity of 93.10 GiB of which 163.88 MiB is free. Process 2176579 has 45.81 GiB memory in use. Including non-PyTorch memory, this process has 47.11 GiB memory in use. Of the allocated memory 46.19 GiB is allocated by PyTorch, and 272.40 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)",
    "pred": {},
    "gold": {},
    "status": "Model reinitialization error",
    "violated_constraint": {},
    "is_exact_match": false,
    "constraints_satisfied": false,
    "pass_number": 1
}